### ISSUE TITLE
App crashes on d.StartUtt()

### ISSUE THREAD

**Issue by chubillkelvin** [Funnyguy77/PocketSphinxUnityDemo#12]
Sorry it's me again.

After debugging for so long, I figured that there was one and only one issue left.

This particular line is causing my app to crash on start: 
```C#
// Create a new decoder with our configuration.
        d = new Decoder(c);
```

Except that I have decompressed and move the language model into local dataPath, everything else is just the same as the sample script given in this project, and I have built my app as an `.apk` to Oculus Quest.

I have tested the project locally in the editor and it works fine. It just causes a crash in Oculus Quest somehow.

If I comment out this particular line, everything works fine on Oculus Quest (of course, the voice recognition won't work but the app at least is working).

Any idea or anyone bumped into the same issue as I do here?

**Comment by Ligul** [comment]
Hi!
I faced the same problem: in the editor and on the Windows platform, everything works fine, and the android build crashes on initialization.
I found out that this only happens when using a keyword file (keywords.txt). Logcat showed that this was due to a file access error, so I tried to move the file to `Application.persistentDataPath` before using it and it solved the problem.
Perhaps my solution will help you too :)
